apps/google-analytics-4/frontend/src/components/main-app/ChartHeader.tsx

```diff
@@ -8,6 +8,15 @@ interface Props {
   handleChange: Function;
 }
 
+const getMetricDisplayString = (_metricName: string) => {
+  switch (_metricName) {
+    case 'screenPageViews':
+      return 'Total Views';
+    default:
+      return 'Undetermined metric';
+  }
+};
+
```

Comment:
Nice this is great. If we were to ever allow the user to switch metrics, could see this existing as a separate object that lives as a constant i.e: `{ screenPageViews: 'Total Views' } `etc. But this is perfect for now 👍